CMakeLists.txt

```diff
@@ -152,20 +155,29 @@ set_target_properties(
     util
     xml
 
-    java_bytecode
-    java-models-library
-    jbmc
-    jbmc-lib
-    janalyzer
-    janalyzer-lib
-    jdiff
-    jdiff-lib
-    java-testing-utils
-    java-unit
-    miniz
-
     PROPERTIES
     CXX_STANDARD 11
     CXX_STANDARD_REQUIRED true
     XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY "Developer ID Application: Daniel Kroening"
 )
+
+if(WITH_JBMC)
```

Comment:
Instead of doing this, just make the `add_subdirectory` controlled by the option and always set these properties in `jbmc/CMakeLists.txt`?